engine/battle/ai/move.asm

```diff
@@ -92,7 +92,7 @@ AIChooseMove: ; 440ce
 
 	push bc
 	ld d, BANK(TrainerClassAttributes)
-	predef FlagPredef
+	predef Predef_Flag
```

Comment:
`FlagActionPredef` would be more clear, and matches `FlagAction` and `EventFlagAction`.